OWML.Common/SocketMessage.cs

```diff
@@ -7,7 +7,8 @@ namespace OWML.Common
 {
     public class SocketMessage
     {
-        public string Sender;
+        public string SenderName;
+        public string SenderFile;
```

Comment:
You get this string from `Method.DeclaringType.Name`, so maybe it should be called `SenderType` instead?

Reply:
done